tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -1635,14 +1692,19 @@ static __always_inline struct ipv6_pinfo *inet6_sk_own_impl(struct sock *__sk, s
     return sk_fullsock ? pinet6_own_impl : NULL;
 }
 
+static inline bool ipv6_any_addr(const struct in6_addr *a)
+{
+    return (a->in6_u.u6_addr32[0] | a->in6_u.u6_addr32[1] | a->in6_u.u6_addr32[2] | a->in6_u.u6_addr32[3]) == 0;
+}
+
 static __always_inline int get_network_details_from_sock_v6(struct sock *sk, net_conn_v6_t *net_details, int peer)
 {
-    struct inet_sock *inet = inet_sk(sk);
+    struct inet_sock *inet = (struct inet_sock *)(sk);
     struct ipv6_pinfo *np = inet6_sk_own_impl(sk, inet);
 
     struct in6_addr addr = {};
     addr = get_sock_v6_rcv_saddr(sk);
-    if (ipv6_addr_any(&addr)){
+    if (ipv6_any_addr(&addr)){
```

Comment:
This function, and inet_sk as well were part of the headers which are now removed for CO-RE, right?
I think that it would be better to put them in the "missing_macros" file (rename?), which should compensate for all the missing things that we had in the headers